lib/apiservers/portlayer/swagger.json

```diff
@@ -2654,8 +2654,8 @@
 			],
 			"properties": {
 				"Stores": {
-					"type": "object",
-					"additionalProperties": {
+					"type": "array",
```

Comment:
Var rename sounds good.  Will do.  The implicit type'ing going on in that function is bugging me.  This *should* have broken during build but because of the use of `:=`, it didn't.